void tomo_db0::IoLoop::r7_run_loop<true, false, false, true, false>()
--- PRE

+++ POST

@@ -502,7 +502,7 @@

 098d [ 3] nopl (%rax)
 0990 [ 5] mov $0x1,%esi
 0995 [ 3] mov %rbx,%rdi
-0998 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0998 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 099d [ 5] jmp <+0x31b>
 09a2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 09ad [ 3] nopl (%rax)
